src/Core/src/Primitives/GridLength.cs

```diff
@@ -120,20 +120,42 @@ public override string ToString()
 		/// <returns><see langword="true"/> if the two GridLengths differ; otherwise, <see langword="false"/>.</returns>
 		public static bool operator !=(GridLength left, GridLength right) => !(left == right);
 
-		private sealed class GridLengthTypeConverter : TypeConverter
+		public sealed class GridLengthTypeConverter : TypeConverter
```

Comment:
as this is becoming public, I wonder if it should level up in the Converters namespace instead of staying a nested type.